### PR TITLE
Insert EasyCatalog syntax

### PR DESCRIPTION
**Pull request by julux**
Hello,
It's possible to add EasyCatalog Syntax?
Regards.

### CODE REVIEW

File: channel.json [hunk 1]
```diff
@@ -59,6 +59,7 @@
 		"https://raw.githubusercontent.com/jfromaniello/sublime-unity-recents/master/packages.json",
 		"https://raw.githubusercontent.com/joacodeviaje/aftersave/master/packages.json",
 		"https://raw.githubusercontent.com/joomlapro/joomla3-sublime-snippets/master/packages.json",
+		"https://raw.githubusercontent.com/julux/EasyCatalog/master/packages.json",
```
Comment (FichteFoll) [inline channel.json]: Please remove this line,

**Comment by FichteFoll** [comment]
You have to push a valid [semantic version](http://semver.org) [tag](http://git-scm.com/book/en/v2/Git-Basics-Tagging) (with optional leading `v`) to your repo.

**Comment by julux** [comment]
Hi,
i do it. 
Regards.

**Comment by julux** [comment]
Hi,
Is it ok? or i did another errors?
I change schema_version in package.json, it's right?
Best regards.

**Comment by FichteFoll** [comment]
I can't see anything. There needs to be a tag on this page: https://github.com/julux/EasyCatalog/tags

Your `package.json` is irrelevant. You can remove it from the repo since it won't be looked at.

**Comment by julux** [comment]
Hi, thank you for your support.
I create tag version and remove package.json from master.
When i create a new release, Package control check the version if the user do the update ?
Regards.

**Comment by FichteFoll** [comment]
Yes, Package Control will automatically find new versions (with little delay).

However, [semantic versions](http://semver.org) consist of exactly three numeric parts.

**Comment by julux** [comment]
hi,
why isn't merge?
regards

**Comment by FichteFoll** [comment]
Github doesn't notify me when commits are appended to a pull request.

**Comment by julux** [comment]
thank you. 
Regards.

**Pull request merged**